src/foundations/shape/Shape.tsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 
 import { Utils } from '../../shared';
+import { DEFAULT_ELEVATION_LEVEL } from '../../utilities/elevation';
 
 import ShapeRoot from './Shape.root';
 import PShape from './Shape.props';
 import {
     DEFAULT_SHAPE_BORDER_RADIUS,
     DEFAULT_SHAPE_ELEMENT,
-    DEFAULT_SHAPE_ELEVATION_LEVEL,
     SHAPE_ELEMENTS,
 } from './Shape.constants';
 
 const Shape = ({
     element = DEFAULT_SHAPE_ELEMENT,
     radius = DEFAULT_SHAPE_BORDER_RADIUS,
-    elevation = DEFAULT_SHAPE_ELEVATION_LEVEL,
-    elevationOnHover = elevation,
+    elevation = DEFAULT_ELEVATION_LEVEL,
```

Comment:
this is component scoped, so it should live in the components constants! 
You can still add it to the components constants file from the elevation constants file and export it under a new name:

`export { ... , DEFAULT_ELEVATION_LEVEL as DEFAULT_SHAPE_ELEVATION_LEVEL, ... };`

Reply:
gotcha! That's a good rule to add to our style guide :)